packages/utils/odsp-doclib-utils/src/odspAuth.ts

```diff
@@ -50,14 +50,12 @@ export function getFetchTokenUrl(server: string): string {
 }
 
 export function getLoginPageUrl(
-    isPush: boolean,
```

Comment:
@AndreiZe this is the function from the build error - you removed one of the parameters 😋

Reply:
Thanks for pointing this out! I've been looking at wrong place before